tools/pipeline_perf_test/orchestrator/requirements.txt

```diff
@@ -1,6 +1,15 @@
-docker
-psutil
-requests
-kubernetes
-pyyaml
-opentelemetry-proto
\ No newline at end of file
+# Dependencies for the pipeline performance test orchestrator
+
+# General dependencies
+requests>=2.25.0
+pyyaml>=5.4.1
+
+# Docker management
```

Comment:
I expect to replace the current spawning cmdline process to run command with these libraries instead. They are much more powerful and easy to use, but started with simple command line tool (as I was running these commands by-hand a lot in parallel!)